src/PKSim.Core/CoreConstants.cs

```diff
@@ -1330,6 +1330,8 @@ public static class UI
          public const int EXPRESSION_QUERY_VIEW_WIDTH = 1100;
          public const int FORMULATION_VIEW_WITDH = 650;
          public const int FORMULATION_VIEW_HEIGHT = 760;
+         public const int OBSERVER_VIEW_HEIGHT = 760;
+         public const int OBSERVER_VIEW_WITDH = 650;
```

Comment:
why are some consts here defined as `static readonly` and some as `const`?

Reply:
Const: They can become default value in function and their value replaces the constant at build time. Static readonly, they cannot be changed but value is not replaced by constant at build time=>cannot be used as default value